### ISSUE TITLE
Player model render armor conflict with vic modern warfare

### ISSUE THREAD

**Issue by Phan7om012** [Drillgon200/Hbm-s-Nuclear-Tech-GIT#53]
Can you pls tell me how to get rid of this annoying conflict, i love both mods but the vic modern warfare mod is breaking hbm's armor render.

![2021-02-21_20 42 27](https://user-images.githubusercontent.com/74053512/108638273-4eb73000-7486-11eb-9e02-84db99af3a47.png)

**Comment by Drillgon200** [comment]
Yeah it does, due to an annoying way in which their coremod works. I can fix it I think, but that also means I have to update every armor class and that's annoying.

**Comment by Phan7om012** [comment]
I know it is going to be annoying but if you can do little fixes from time to time to the other updates you will announce, I wil be very thankfull and the pepole who are using both this mods, I know its difficult and I understand how hard coding is and I really appreciate your work because its not easy to make a massive mod like this but pls maybe you will find a easier way to fix it :>

**Comment by Drillgon200** [comment]
It's not even hard it's just tedious.

**Comment by Phan7om012** [comment]
If you will find a time to do it pls fix the problem
